mypy/util.py

```diff
@@ -3,6 +3,7 @@
 import re
 import subprocess
 import hashlib
+from io import BytesIO
```

Comment:
I'd prefer the style `import io` here (in general for stdlib modules, though I'm fine with `escape` below).